emailvalidation/emailvalidation.go

```diff
@@ -0,0 +1,25 @@
+package emailvalidation
```

Comment:
This feels very very specific. Maybe we could broaden it a bit and make this a `validation` package? That way it will serve a specific purpose (validation) but won't be so specific it is only used in 1 case.

Reply:
I think there was the feeling that it might be worth having a specific package for email validation, because whilst specific it would be reused in more than one scenario?

Reply:
This package has been updated to validation